Automattic-Tracks-iOS/TracksService.m

```diff
@@ -116,7 +118,7 @@ - (void)trackEventName:(NSString *)eventName withCustomProperties:(NSDictionary
                                               username:self.username
                                                 userID:self.userID
                                              userAgent:self.userAgent
-                                              userType:self.isAnonymous ? TracksEventUserTypeAnonymous : TracksEventUserTypeWordPressCom
+                                              userType:self.isAnonymous ? TracksEventUserTypeAnonymous : TracksEventUserTypeAuthenticated
```

Comment:
userType parameter is an NSInteger. That gets converted, down the line, into a NSString (by means of the new two properties!)